test/chat_test.py

```diff
@@ -77,6 +85,42 @@ def test_send_chat_message(self):
 
         self.disconnect_users([self.user4, self.user3, self.user1])
 
+    def test_chat_offline_message(self):
+        print("== Test offline chat messages")
+        firs_offline_message = "first offline message"
+        second_offline_message = "second offline message"
+        self.user4 = self.getLoggedInUser(self.name_pass4)
+        time.sleep(2)
+        print("Send first message")
+
+        self.user4.send_message(
```

Comment:
It's better to create messages with constructor from protocol.messages and then call it's method 'as_bytes'